hummingbird/ml/convert.py

```diff
@@ -149,14 +149,18 @@ def _convert_onnxml(model, backend, test_input, device, extra_config={}):
             ), "Cannot generate test input data. Initial_types do not contain shape information."
             assert len(initial_types[0][1].shape) == 2, "Hummingbird currently support only inputs with len(shape) == 2."
 
-            from onnxconverter_common.data_types import FloatTensorType, Int32TensorType
+            from onnxconverter_common.data_types import FloatTensorType, DoubleTensorType, Int32TensorType, Int64TensorType
 
             test_input = np.random.rand(initial_types[0][1].shape[0], initial_types[0][1].shape[1])
             extra_config[constants.N_FEATURES] = initial_types[0][1].shape[1]
             if type(initial_types[0][1]) is FloatTensorType:
                 test_input = np.array(test_input, dtype=np.float32)
+            elif type(initial_types[0][1]) is DoubleTensorType:
```

Comment:
please add a test case for this and lines 162.

(Trying to keep coverage above 92% if possible!)

Reply:
Added.